examples/browser/counter.html

```diff
@@ -44,7 +44,7 @@ <h1>Counter Client Example in the Browser</h1>
                     readonly="readonly"
                     id="td_addr"
                     type="url"
-                    value="http://plugfest.thingweb.io:8083/counter"
+                    value="http://localhost:8080/counter"
```

Comment:
Do we want to keep the local or online counter instance?

Reply:
Right sorry, it was a leftover from local tests.